config/ironfurnaces-client.toml

```diff
@@ -0,0 +1,277 @@
+
+#Furnace Settings
+[furnaces]
+	# The capacity of the recipe cache, higher values use more memory.
+	# Default: 10
+	#Range: 1 ~ 100
+	recipe_cache = 10
+
+	[furnaces.energy]

```

Comment:
this looks like serverside configs on client?

Reply:
this is unchanged by me

Reply:
oh nvm for some reason ironfurnaces just use server configs